### DISCUSSION TITLE
How to have Tags

### DISCUSSION THREAD

**Discussion by Dav-d** [the-djmaze/snappymail#1316] [Q&A]
Hi,

I see in the demo the tag (label) feature. But I don't have this on my version snappymail. I use the 2.29.2 version with nextcloud (21.0.9). I didn't see any extension in the nextcloud snappymail extension list.
So, What is the name of the extension I have to activate if any, Or how to do to have the tag feature ?

Thanks a lot

**Comment by cm-schl** [comment]
Hi,

you can tag mails by using the little symbol on the right side of the mail header:
![grafik](https://github.com/the-djmaze/snappymail/assets/63400209/556adf2a-ab00-4eb2-85a1-a2e7039379bc)

Hope that helps 🙂

**Comment by Dav-d** [comment]
Hi, thank you for you reply.

But I'm sorry because there is no tag icon in the mail header, no more than there is a tag option in the search menu...

Regards

**Reply by cm-schl** [reply]
strange... I don't know if there is some setting to activate / deactivate tags, but as far as I know an extension is not needed.
You could try to change the skin of Snappymail as maybe this is "just" a problem in the visualization 🤷‍♂️

**Reply by Dav-d** [reply]
nope... I tried with other themes (default, nextcloud, nextcloud25, wood...) but none works.
But now, I know that no extension is needed. Thank you :) 
Maybe this issue is due to my old version of nextcloud...

**Comment by Dav-d** [comment]
Same issue with Nextcloud 25.0.13 and after delete and reinstall snappymail.

**Comment by the-djmaze** [comment]
Your IMAP server does not support the `\*` permanent flag.
Change to a different server.

**Comment by botsarenthuman** [comment]
How do you search for multiple tags, using `OR` ?
